### PR TITLE
Fixes the bee dilemma

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
add: A new solution to our bee problem
/:cl:

https://streamable.com/eqdty

4thlusv

### CODE REVIEW

**Comment by Jordie0608** [comment]
This is a step in an orthogonal direction.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -94,6 +94,26 @@
...
+	item_color = "clown"
+	var/footstep = TRUE	//used for squeeks whilst walking
+
+/obj/item/clothing/shoes/bees/step_action()
+	if(footstep)
+		playsound(src, 'sound/effects/fuck.ogg', 50, 1)
+		footstep = !footstep
+	else
+		playsound(src, 'sound/effects/bees.ogg', 50, 1)
+		footstep = !footstep
```
Comment (Cyberboss) [inline code/modules/clothing/shoes/miscellaneous.dm]: Dupe code

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/clothing/shoes/miscellaneous.dm]: Yeah just move the !footstep out to below the if

**Review by Cyberboss — APPROVED** [review]

**Comment by Qustinnus** [comment]
added some sprites

**Comment by Cyberboss** [comment]
As good as this is, no

**Comment by optimumtact** [comment]
1 hour remains

**Comment by lzimann** [comment]
what he said

**Comment by Iamgoofball** [comment]
Why

**Comment by Iamgoofball** [comment]
Literally nothing wrong with this PR

**Pull request closed without merging**